ilc/Dockerfile

```diff
@@ -1,14 +1,32 @@
-FROM node:12-alpine
+FROM node:12-alpine AS build
 
-RUN apk update && apk add --no-cache bash git openssh python make g++ findutils
+RUN apk update && apk add --no-cache bash git openssh python make g++ findutils chromium chromium-chromedriver
 
-ADD ./ /codebase
-WORKDIR /codebase
+ENV CHROME_BIN=/usr/bin/chromium-browser
+ENV CHROME_DRIVER=/usr/bin/chromedriver
+
+ADD ./ /temporary
+WORKDIR /temporary
 
 RUN npm install --no-package-lock --no-progress
+RUN npm run test
```

Comment:
I don't really like the approach of running tests within docker build process as it makes impossible to extract test artefacts & attach them as CI run results

Reply:
Look at the https://docs.docker.com/engine/reference/commandline/build/#specifying-target-build-stage---target

Reply:
Agree, In the multistage building it is impossible to fetch artifacts from the not final stage. --target can solve the problem but then it's required to solve the "copy" problem. By "copy" problem I mean either to run a container and copy data OR use --out option for the build command but it will copy all the filesystem to the host.

Probably, in this case, it's better to split into two docker files. The first one with chrome, we build, create a container, and run the tests with the mounted volume (to get the artifacts on the host). 
And the second one is clean Dockerfile without test dependencies (which is the main goal as I understand)